portable/CMakeLists.txt

```
@@ -182,6 +182,25 @@ add_library(freertos_kernel_port OBJECT
         GCC/ARM_CM55_NTZ/non_secure/mpu_wrappers_v2_asm.c
         ThirdParty/GCC/ARM_TFM/os_wrapper_freertos.c>
 
+    $<$<STREQUAL:${FREERTOS_PORT},GCC_ARM_CM52_NONSECURE>:
+        GCC/ARM_CM52/non_secure/port.c
+        GCC/ARM_CM52/non_secure/portasm.c
+        GCC/ARM_CM52/non_secure/mpu_wrappers_v2_asm.c>
+    $<$<STREQUAL:${FREERTOS_PORT},GCC_ARM_CM52_SECURE>:
```

Can you please add an empty line to align with the existing structure?
```suggestion

    $<$<STREQUAL:${FREERTOS_PORT},GCC_ARM_CM52_SECURE>:
```